locales/de-DE.json

```diff
@@ -18,29 +26,44 @@
     "joined": "Beigetreten",
     "moved_title": "hat angegeben, dass dies der neue Account ist:",
     "muted_users": "Stummgeschaltete Accounts",
+    "muting": "Stummgeschaltet",
     "mutuals": "Freunde",
+    "notify_on_post": "Benachrichtigen Sie mich, wenn {username} etwas postet",
     "pinned": "Angepinnt",
     "posts": "Beiträge",
     "posts_count": "{0} Beiträge",
     "profile_description": "{0}'s Profil",
     "profile_unavailable": "Profil nicht verfügbar",
-    "unfollow": "Entfolgen"
+    "unblock": "Entsperren",
+    "unfollow": "Entfolgen",
+    "unmute": "Stummschaltung aufheben",
+    "view_other_followers": "Follower aus anderen Instanzen werden möglicherweise nicht angezeigt.",
```

Comment:
Folgende? or do we use "follower"  consistently in the translation

Reply:
I think this is correct as is